run_tests/const.ts

```diff
@@ -0,0 +1,29 @@
+import path from "path";
+
+/**
+ * Directory containing all test directories
+ */
+export const exampleFoldersDir = path.parse(__dirname).dir;
+
+/**
+ * Port used by static server
+ */
+export const serverPort = 3000;
+
+/**
+ * Directory names of tests that aren't supported by this automation script
```

Comment:
Reference Readme section with explanations. (And from README reference this location as to where tests are skipped?)

Reply:
good idea! addressed this in https://github.com/WebBluetoothCG/manual-tests/pull/39/commits/6d020a4d03b4cb96f7a6070ba9cccd3f72734b7c